src/roswire/proxy/roslaunch/reader.py

```diff
@@ -2,11 +2,13 @@
 """
 This file implements a proxy for parsing the contents of launch files.
 """
-__all__ = ('LaunchFileReader',)
+__all__ = ('ROS1LaunchFileReader',)
```

Comment:
This also exports the `LaunchFileReader` interface.

Reply:
Done.